package.json

```diff
@@ -51,16 +56,27 @@
     "recharts": "^2.2.0"
   },
   "devDependencies": {
+    "@babel/core": "^7.20.7",
     "@parcel/packager-raw-url": "^2.8.1",
     "@parcel/transformer-webmanifest": "^2.8.1",
+    "@storybook/addon-actions": "^6.5.15",
+    "@storybook/addon-essentials": "^6.5.15",
+    "@storybook/addon-interactions": "^6.5.15",
+    "@storybook/addon-links": "^6.5.15",
+    "@storybook/builder-webpack4": "^6.5.15",
+    "@storybook/manager-webpack4": "^6.5.15",
+    "@storybook/react": "^6.5.15",
+    "@storybook/testing-library": "^0.0.13",
     "@types/node": "^18.11.13",
     "@types/react": "^18.0.26",
     "@types/react-dom": "^18.0.9",
+    "babel-loader": "^8.3.0",
```

Comment:
Why we need this ? Doesn't storybook use ts-loader under the hood?

Reply:
It is actually a peer-dependency of webpack loader, so I guess not:

```
yarn why v1.22.19
[1/4] 🤔  Why do we have the module "babel-loader"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "babel-loader@8.3.0"
info Has been hoisted to "babel-loader"
info Reasons this module exists
   - Specified in "devDependencies"
   - Hoisted from "@storybook#builder-webpack5#babel-loader"
   - Hoisted from "@storybook#manager-webpack5#babel-loader"

```

Reply:
ah so yeah I guess we don't need that in our package.json